src/test/features/claim/form/models/claimAmount.ts

```diff
@@ -217,7 +217,7 @@ describe('ClaimAmountBreakdown', () => {
     })
   })
 
-  describe('totalAmount', () => {
+  describe('totalAmountTillToday', () => {
```

Comment:
Please revert it. Refactoring changed it automatically but the test is for `Breakdown#totalAmount` not for the `Claim#totalAmount`